mapDownload.properties

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I agree about the location, but I have trouble thinking of a much better one. Likely when we are ready to split things apart: dice roller, lobby, client; that could be a good time to move the config file.

But by the same token, a reason to keep it in the main repo, map makers can use the github web UI to create a fork/branch and PR to update this file. We'll want to see those PRs. The map project repo could be a good place, but I think that is likely better left as a "quiet" repo that stores our map admin scripts and documentation (and namely not a critical configuration file like this)